### PR TITLE
Add --ignore-missing-imports flag to mypy run command.

### PR DESCRIPTION
**Pull request by hramezani**

### CODE REVIEW

File: src/urllib3/util/url.py [hunk 1]
```diff
@@ -300,7 +300,7 @@ def _normalize_host(
 def _idna_encode(name: str) -> bytes:
     if name and any([ord(x) > 128 for x in name]):
         try:
-            import idna  # type: ignore
+            import idna
```
Comment (sethmlarson) [inline src/urllib3/util/url.py:303]: Doesn't the latest idna include type hints?

**Review by sethmlarson — COMMENTED** [review]
I'm unsure about this one, I think I prefer the explicit `type: ignore` on third party modules without type hints?

File: src/urllib3/util/url.py (hunk 1 above)
Comment (hramezani) [reply src/urllib3/util/url.py:303]: I think it does. but we need to install it in `idna` in the `lint` environment.

Comment (sethmlarson) [reply src/urllib3/util/url.py:303]: Let's do that!

Comment (hramezani) [reply src/urllib3/util/url.py:303]: Ok, I will create a new PR for this.

**Pull request closed without merging**